### PR TITLE
Bug 1855577: Updating the golang.org/x/text version to v0.3.3

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
With this change

**Comment by LalatenduMohanty** [comment]
There are indirect dependency on `golang.org/x/text` , so I am not sure how to resolve this elegantly.  
for example 
```
$ go list -m all | grep 'golang.org/x/text'
golang.org/x/text v0.3.3
```

**Comment by LalatenduMohanty** [comment]
Another approach https://github.com/openshift/cluster-version-operator/pull/406/. However we need to merge the fix asap.

File: go.mod [hunk 1]
```diff
@@ -25,6 +25,7 @@ require (
 	github.com/spf13/cobra v0.0.5
 	golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550
 	golang.org/x/sys v0.0.0-20191003212358-c178f38b412c // indirect
+	golang.org/x/text v0.3.3 // indirect
```
Comment (vrutkovs) [inline go.mod:28]: `replace` would ensure other packages won't overwrite that

**Pull request merged**